test/active_record_test.rb

```diff
@@ -366,9 +366,28 @@ class Measured::Rails::ActiveRecordTest < ActiveSupport::TestCase
     assert_equal Measured::Length.new(500, :mm), thing.length_with_max_on_assignment
   end
 
-    test "assigning a small number to a field that specifies max_on_assignment" do
-    thing = Thing.create!(length_with_max_on_assignment: Measured::Length.new(1, :mm))
-    assert_equal Measured::Length.new(1, :mm), thing.length_with_max_on_assignment
+  test "assigning a small number to a field that specifies max_on_assignment" do
+     thing = Thing.create!(length_with_max_on_assignment: Measured::Length.new(1, :mm))
+     assert_equal Measured::Length.new(1, :mm), thing.length_with_max_on_assignment
+  end
+
+  test "using a similar unit accessor for multiple size fields" do
+    assert_equal custom_unit_thing.length, Measured::Length.new(1, :m)
```

Comment:
Can we swap assertion to: 

`assert_equal Measured::Length.new(1, :m), custom_unit_thing.length`